### PR TITLE
Move documentation out of Akri's main repository [SAME VERSION]

### PR DESCRIPTION
**Pull request by kate-goldenring**
**What this PR does / why we need it**:
Akri's documentation has been moved to a separate repository [deislabs/akri-docs](https://github.com/deislabs/akri-docs). This removes all of the documentation and media that has been transferred to that repository. Namely:
- Documentation under `akri/docs/` was either moved to `akri-docs/docs`, `akri-docs/proposals`, or Akri's [HackMD](https://hackmd.io/team/akri?nav=overview)
- Image files under `akri/docs/media` were moved to `akri-docs/media`
- Logo files under `akri/art` were moved to `akri-docs/art`

Also updates README and Pull Request template to point to Akri's documentation on it's documentation site [docs.akri.sh](https://docs.akri.sh/).

**Special notes for your reviewer**:
With this PR will commence the convention to put documentation for a PR in a separate PR into our documentation repository, which is noted in the PR template.
Note: README points to Raspberry Pi demo endpoint that will exist once https://github.com/deislabs/akri-docs/pull/1 is merged

### CODE REVIEW

**Comment by bfjelds** [comment]
this looks good to me ... the only other link to docs i see in my repo is for gh-pages' index.md (https://github.com/deislabs/akri/blob/gh-pages/index.md)

**Comment by jiria** [comment]
FYI the PR that you linked for rpi demo is merged, yet the rpi demo url is still not working. https://docs.akri.sh/demos/usb-camera-demo-rpi4

**Comment by kate-goldenring** [comment]
> FYI the PR that you linked for rpi demo is merged, yet the rpi demo url is still not working. https://docs.akri.sh/demos/usb-camera-demo-rpi4

Yeah that is because the site is still pointing at my fork. I need to get appropriate permissions from Deis Labs folks to switch it, so this PR is one hold until that happens

**Comment by kate-goldenring** [comment]
> this looks good to me ... the only other link to docs i see in my repo is for gh-pages' index.md (https://github.com/deislabs/akri/blob/gh-pages/index.md)

@bfjelds good point. Put in a PR to update `gh-pages` too: https://github.com/deislabs/akri/pull/362

File: deployment/helm/templates/udev-configuration.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ spec:
     name: udev
     discoveryDetails: |+
       udevRules:
-      {{- required "Please set at least one udev rule with `--set udev.configuration.discoveryDetails.udevRules[0]==\"<udev rule>\"' to specify what you want discovered. See the udev Configuration document at https://github.com/deislabs/akri/blob/main/docs/udev-configuration.md for more information." .Values.udev.configuration.discoveryDetails.udevRules | toYaml | nindent 6 }}
+      {{- required "Please set at least one udev rule with `--set udev.configuration.discoveryDetails.udevRules[0]==\"<udev rule>\"' to specify what you want discovered. See the udev Configuration document at https://docs.akri.sh/discovery-handlers/udev for more information." .Values.udev.configuration.discoveryDetails.udevRules | toYaml | nindent 6 }}
```
Comment (kate-goldenring) [inline deployment/helm/templates/udev-configuration.yaml:11]: This change to the template is what is kicking off the end to end tests. I think it makes sense to still not do a version update. This log will be updated in a later chart.

**Review by jiria — APPROVED** [review]

**Pull request merged**